rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -10,8 +10,7 @@ categories = ["asynchronous", "web-programming", "api-bindings"]
 description = """
 Server runtime for Smithy Rust Server Framework.
 """
-# until this is not stable, it is not publishable.
-publish = false
+publish = true
```

Comment:
This should be done also for `aws-smithy-http-server-python`

Reply:
Done. Have changed the Cargo.toml for python to be auto published as well.

Reply:
If you want it to be included in the automated smithy-rs releases, it will need to be added to this list as well: https://github.com/awslabs/smithy-rs/blob/b77c66cbf72e8a6083af87cf125fd7d84eaa69a4/buildSrc/src/main/kotlin/CrateSet.kt#L34